CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.6
+
+<!-- generated comparing v5.14.5..master -->
+
+_Aug 23, 2023_
+
+A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Added [Accordion](https://mui.com/joy-ui/react-accordion/) component to Joy UI (#38164) @siriwatknp
+- 🎉 Added Material You Input and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert
```

Comment:
```suggestion
- 🎉 Added Material You InputBase and ButtonBase components to `material-next` (#38319) @DiegoAndai @mj12albert
```

It should be `InputBase` and not `Input` ~ (which is the standard variant Input)

Reply:
Done!